qiskit_experiments/library/tomography/tomography_analysis.py

```diff
@@ -110,21 +110,40 @@ def _run_analysis(self, experiment_data):
 
         # Get tomography fitter function
         fitter = self._get_fitter(self.options.fitter)
+        fitter_opts = self.options.fitter_options
+
+        # Work around to set proper trace and trace preserving constraints for
+        # cvxpy fitter
+        if fitter in (cvxpy_linear_lstsq, cvxpy_gaussian_lstsq):
```

Comment:
Can we extend this function for these options rather than handling them here? Probably difficult because of the missing data size?
https://github.com/Qiskit/qiskit-experiments/blob/d690a89ee59c1c564bad7f2812b890955b42d2d8/qiskit_experiments/library/tomography/fitters/cvxpy_utils.py#L111-L118